### PR TITLE
Fixes applied during the September Maintenance Tasks

### PR DESCRIPTION
**Pull request by adgiles**
Addressed GitHub Issues:

https://github.com/ArcGIS/solutions-desktop-templates/issues/169
https://github.com/ArcGIS/solutions-desktop-templates/issues/262
https://github.com/ArcGIS/solutions-desktop-templates/issues/267

### CODE REVIEW

**Comment by csmoore** [comment]
@mfunk - just let me know if we should hold off on this PR until we have the meeting to discuss this repo. For example - in this PR's case - if it is OK to combine the Pro & ArcMap toolboxes if the ArcMap one can be used as-is in Pro. Otherwise, I'll merge in a day or so if I don't hear back.

**Comment by mfunk** [comment]
@adgiles @csmoore We will not merge toolboxes for several reasons:
* Tool edits in Pro are NOT backwards compatible with ArcMap (any version) or earlier versions of Pro. Read **Note** under *Compatibility* section: http://pro.arcgis.com/en/pro-app/help/analysis/geoprocessing/modelbuilder/modelbuilder-changes-in-arcgis-pro.htm
* System tools are different between Map and Pro (Pro is getting new tools that don't exist in Map)

Yes, this sucks and is not a nice solution, but it is the method we decided to use to solve the problem at hand. Until ArcGIS Desktop (ArcMap/ArcCatalog/ArcScene/ArcGlobe) disappears completely and forever... then we will revisit this.

**Comment by csmoore** [comment]
Just to try to clarify - the suggested change to this PR is to add a file:

operational_graphics/toolboxes/`ClearingOperationsTools_10.3.tbx`  (or 10.4 - pending meeting to discuss version supported by this repo vs. templates) 

This is a bit off-topic but would Pro, like most products, warns you when you are about to save your toolbox to an older/incompatible format - does this help address this editing scenario mentioned? 

image

**Comment by mfunk** [comment]
@csmoore There are toolboxes for ArcMap and Pro in the repo already:
* https://github.com/Esri/solutions-geoprocessing-toolbox/blob/dev/operational_graphics/toolboxes/Clearing%20Operations%20Tools_10.4.tbx
* https://github.com/Esri/solutions-geoprocessing-toolbox/blob/time-selection/operational_graphics/toolboxes/Clearing%20Operations%20Tools.tbx

The PR changes removes both of these and replaces it with one TBX. This is not how we are handling the toolboxes for ArcGIS Desktop and ArcGIS Pro.

**Comment by mfunk** [comment]
@adgiles Per today's meeting on the repo can you change the toobox names back to _pro.tbx (as 1.4) and _arcmap.tbx (as 10.3) for this PR.

**Comment by adgiles** [comment]
If you haven't merged this change in there is nothing to merge back

**Comment by mfunk** [comment]
As part of this PR i'm still seeing the change that the original TBXs are going to be removed and replaced with a single TBX. We need to keep the separate TBX files, but include any fixes made for template issues 169, 262, and 267.

File: operational_graphics/toolboxes/scripts/PointTargetGRG.py [hunk 1]
```diff
@@ -466,11 +471,11 @@ def main():
         #UPDATE
         toolboxPath = None
         if appEnvironment == "ARCGIS_PRO":
-            toolboxPath = os.path.join(os.path.dirname(sysPath), "Clearing Operations Tools.tbx")
+            toolboxPath = os.path.join(os.path.dirname(sysPath), "ClearingOperationsTools.tbx")
```
Comment (mfunk) [inline operational_graphics/toolboxes/scripts/PointTargetGRG.py]: Will need to update the toolbox name here: **ClearingOperationsTools_pro.tbx**.

File: operational_graphics/toolboxes/scripts/PointTargetGRG.py [hunk 2]
```diff
@@ -466,11 +471,11 @@ def main():
...
         else:
-            toolboxPath = os.path.join(os.path.dirname(sysPath), "Clearing Operations Tools_10.4.tbx")
+            toolboxPath = os.path.join(os.path.dirname(sysPath), "ClearingOperationsTools.tbx")
```
Comment (mfunk) [inline operational_graphics/toolboxes/scripts/PointTargetGRG.py]: Will need to update the toolbox name here: **ClearingOperationsTools_arcmap.tbx**.

**Pull request merged**